pkg/controller/backup_controller_test.go

```diff
@@ -148,6 +149,55 @@ func TestProcessBackupNonProcessedItems(t *testing.T) {
 	}
 }
 
+// OADP Carry: Test that backup that has status inProgress on reconcile is changed to failed if velero has no memory of it still in-progress.
+func TestProcessBackupInProgressFailOnSecondReconcile(t *testing.T) {
+	tests := []struct {
+		name            string
+		key             string
+		tracked         bool
+		reconciledPhase velerov1api.BackupPhase
+		expectedErr     error
```

Comment:
this is not used, right? would remove it

Reply:
```suggestion
```

Reply:
actually, I guess I expected err to be nil.. check against nil is still being checked [here](https://github.com/openshift/velero/pull/324/files#diff-701da9642e59239de4f6bc012f4a307887935cd4414f6b9081d3b0a5bcf3b79dR192).

Reply:
would not be simpler to assert nil like here https://github.com/openshift/velero/pull/324/files#diff-701da9642e59239de4f6bc012f4a307887935cd4414f6b9081d3b0a5bcf3b79dR195 ?

Reply:
Perhaps. If we expect err in the future then readd expectedErr?

Reply:
the tests now has expectedErr so this is resolved.